Editor/UI/ProjectAuditorWindow.cs

```diff
@@ -634,6 +647,8 @@ IssueCategory[] GetSelectedCategories()
                 requestedCategories.AddRange(m_ProjectAuditor.GetModule<AssetsModule>().GetCategories());
             if ((m_SelectedModules.HasFlag(BuiltInModules.BuildReport)))
                 requestedCategories.AddRange(m_ProjectAuditor.GetModule<BuildReportModule>().GetCategories());
+            if ((m_SelectedModules.HasFlag(BuiltInModules.Texture)))
```

Comment:
I think we don't want to expose this yet

Reply:
Removed in [Tweaks to Texture Module] commit.